pkg/quota/v1/evaluator/core/persistent_volume_claims.go

```diff
@@ -90,26 +94,73 @@ func (p *pvcEvaluator) GroupResource() schema.GroupResource {
 
 // Handles returns true if the evaluator should handle the specified operation.
 func (p *pvcEvaluator) Handles(a admission.Attributes) bool {
-	if a.GetSubresource() != "" {
+	op := a.GetOperation()
+	switch a.GetSubresource() {
+	case "":
+		return op == admission.Create || op == admission.Update
+	case "status":
```

Comment:
Is there any other status field of other objects will get impacted by this change?

Reply:
For resizer recovery, the current behavior is that the status change is impacted by the admission quota?

Reply:
@sunnylovestiramisu https://github.com/carlory/kubernetes/blob/43040290a30bb4d59499d8e0f5833a28b8510de2/staging/src/k8s.io/apiserver/pkg/quota/v1/interfaces.go#L52